SU2_CFD/include/iteration/modules/SolverModules.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "CFixedCLModule.hpp"
+
+using Modules    = CModuleList<CConfig, CFixedCLModule>;
```

Comment:
Here we define what modules we actually have (the CConfig is just used to define the argument for the constructor of the modules, can be also moved to the base class of CModuleList.